### PR TITLE
Update instruction to wait after each step

### PR DESCRIPTION
**Pull request by georgebrock**
### Why:

Training new users to refresh after each action seems like an anti-pattern: these actions should apply more or less immediately, and there shouldn't be any need to refresh the page in between (if there is please let us – the PRs team – know so we can fix it!)

### What's being changed:

Removing the “wait about 20 seconds and then refresh the page” instruction from each step.

### Check off the following:

- [ ] ~~For workflow changes, I have verified the Actions workflows function as expected.~~
- [x] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,7 +62,6 @@ The best way to get a review is to ask for one. On GitHub, you can ask someone t
 1. Open a new browser tab, and work on the steps in your second tab while you read the instructions in this tab.
 1. We made a pull request for you from the `update-game` branch, so open that pull request.
 1. Under **Reviewers** on the right side of the screen, add yourself as a reviewer.
-1. Wait about 20 seconds then refresh this page for the next step.
```
Comment (heiskr) [inline README.md:65]: @dihydroJenoxide Updated the copy in https://github.com/skills/hello-github-actions to this to clarify the instruction is meant for the instructions page, not the pull request.

> Wait about 20 seconds for actions to run, then refresh this page (the one you're following instructions from) and an action will automatically close this step and open the next one.

File: README.md [hunk 2]
```diff
@@ -62,7 +62,6 @@ The best way to get a review is to ask for one. On GitHub, you can ask someone t
 1. Open a new browser tab, and work on the steps in your second tab while you read the instructions in this tab.
 1. We made a pull request for you from the `update-game` branch, so open that pull request.
 1. Under **Reviewers** on the right side of the screen, add yourself as a reviewer.
```
Comment (dihydroJenoxide) [inline README.md:64]:
```suggestion
1. Under **Reviewers** on the right side of the screen, add yourself as a reviewer.

Next, wait about 20 seconds for actions to run, then refresh this course's README (the one you're following instructions from) and a [GitHub Action](https://docs.github.com/en/actions) will automatically close this step and open the next one.
```

File: README.md [hunk 3]
```diff
@@ -106,7 +105,6 @@ When an approval or request for changes is not yet needed, consider using **comm
 1. Add a comment with your initial thoughts on the pull request.
 1. Select _comment_, _approve_, or _request changes_: any option will work.
 1. Click **Submit review**.
```
Comment (dihydroJenoxide) [inline README.md:107]:
```suggestion
1. Click **Submit review**.

Next, wait about 20 seconds for actions to run, then refresh this course's README (the one you're following instructions from) and a [GitHub Action](https://docs.github.com/en/actions) will automatically close this step and open the next one.
```

File: README.md [hunk 4]
```diff
@@ -134,7 +132,6 @@ Now that you have explored the different ways you can review a pull request it i
 1. After the commment form appears, click the **Insert a suggestion** button.
 1. Edit the suggestion.
 1. Click **Add a single comment**.
```
Comment (dihydroJenoxide) [inline README.md:134]:
```suggestion
1. Click **Add a single comment**.

Next, wait about 20 seconds for actions to run, then refresh this course's README (the one you're following instructions from) and a [GitHub Action](https://docs.github.com/en/actions) will automatically close this step and open the next one.
```

File: README.md [hunk 5]
```diff
@@ -156,7 +153,6 @@ Now let's see how easy it is to [apply your suggestion](https://docs.github.com/
 1. Click **Commit suggestion**.
 1. Type a commit message.
 1. Click **Commit changes**.
```
Comment (dihydroJenoxide) [inline README.md:155]:
```suggestion
1. Click **Commit changes**.

Next, wait about 20 seconds for actions to run, then refresh this course's README (the one you're following instructions from) and a [GitHub Action](https://docs.github.com/en/actions) will automatically close this step and open the next one.
```

File: README.md [hunk 6]
```diff
@@ -177,7 +173,6 @@ You can now [merge](https://docs.github.com/en/get-started/quickstart/github-glo
 
 1. Click **Merge pull request**.
 1. Delete the branch `update-game` (optional).
```
Comment (dihydroJenoxide) [inline README.md:175]:
```suggestion
1. Delete the branch `update-game` (optional).

Next, wait about 20 seconds for actions to run, then refresh this course's README (the one you're following instructions from) and a [GitHub Action](https://docs.github.com/en/actions) will automatically close this step and open the next one.
```

**Review by dihydroJenoxide — CHANGES_REQUESTED** [review]
I updated all the instances where you deleted the instructions to wait with updated language to reflect the waiting is for GitHub Actions and the instruction page instead of for the feature. 

If you'll accept those changes, or discuss further in Slack, then we can get this approved and merged! 

Feel free to open up a similar PR for the other course your team is concerned about and we'll get it approved. If you don't, [we have an issue](https://github.com/github/skills/issues/58) to update this language for all the courses and we'll get it done. 

cc @mntlty @danhodos

**Review by heiskr — APPROVED** [review]

**Pull request merged**